src/RollbarLogHandler.php

```diff
@@ -75,6 +75,11 @@ public function log($level, $message, array $context = [])
 
         $context = $this->addContext($context);
 
+        // Workaround Laravel 5.5 moving exception from message into context
+        if (! empty($context['exception']) && is_string($message)) {
+            $message = $context['exception'];
+        }
```

Comment:
If `$message` is a string (which you're checking with `is_string()`, then are you truly wanting to overwrite it like this (you used `=`), instead of adding to it (ie: with `.=`)?

Reply:
Yes, I want to replace not append.

If you follow the rabbit hole far enough that "$message" becomes "$toLog" for a "getBody" method that checks whether it is an instance of an exception in order to know whether or not to get a trace.

Reply:
After digging, I agree.
👍